pkg/daemon/client/client.go

```diff
@@ -25,6 +25,8 @@ type Client struct {
 
 // New initializes a new API client
 func New(endpoint string) *Client {
+	fmt.Printf("client initialized against addr: %s\n", endpoint)
```

Comment:
We should be consistent with logging, and stop having `fmt.Println` that people can't enable and disable. DEBUG, INFO, WARN, ERROR gives us that flexibility, no? Also this is not structured, in the future we might want to have tests for important log lines and get some of the events they generate.

Reply:
This particular one should be `DEBUG` in my opinion.

Reply:
For the reason I stated above. This is really _pre-initialisation_/prelude logging. Honestly, I hated doing this, but there's no way I can unconditionally print a log statement with zap. Unless we create a custom logging level? I don't have time to look into that; if you do, be my guest ;-)